### PR TITLE
Bug1845790 reduce 2022 first VM bootup

### PR DESCRIPTION
**Pull request by markcor**

### CODE REVIEW

File: data/roles/win2022642009azurevs.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+
+win-worker:
+  # generic-worker
+  generic_worker:
+    # File versions
+    name: "generic-worker-multiuser-windows-amd64"
+    version: '53.2.1'
```
Comment (jwmoss) [inline data/roles/win2022642009azurevs.yaml:8]: Did you want to bump this to `54.4.1`? That's what I'm rolling out with windows cloud workers (and hardware).

File: modules/roles_profiles/manifests/profiles/disable_services.pp [hunk 2]
```diff
@@ -52,6 +52,10 @@
             if $facts['os']['release']['full'] == '10' {
                 include win_disable_services::disable_onedrive
             }
+            ## 2016 == 2022
+            if $facts['os']['release']['full'] == '2016' {
```
Comment (jwmoss) [inline modules/roles_profiles/manifests/profiles/disable_services.pp:56]: Does this translate to 2022 or 2016? Would it be better to have a custom fact instead?

**Review by jwmoss — CHANGES_REQUESTED** [review]

File: data/roles/win2022642009azurevs.yaml (hunk 1 above)
Comment (markcor) [reply data/roles/win2022642009azurevs.yaml:8]: good call.

File: modules/roles_profiles/manifests/profiles/disable_services.pp (hunk 2 above)
Comment (markcor) [reply modules/roles_profiles/manifests/profiles/disable_services.pp:56]: I am good with leaving it as is for now.  I want to get this out early next week to level 1/2 workers. However, we should refactor to `$facts['custom_win_os_version']` everywhere at some point.

**Review by jwmoss — APPROVED** [review]

**Pull request merged**